### PR TITLE
Removing old wrapper commands

### PR DESCRIPTION
**Pull request by jaygambetta**
This is the fist step in doing #1083. 

Issues i still have is cleaning up the test_results and test_wrapper

Next is the splitting up the transpile.compile and then making dagtocircuits. Then doing parallel.

### CODE REVIEW

File: qiskit/wrapper/_wrapper.py [hunk 1]
```diff
@@ -237,7 +26,7 @@ def compile(circuits, backend,
 
     Args:
         circuits (QuantumCircuit or list[QuantumCircuit]): circuits to compile
-        backend (BaseBackend or str): a backend to compile for
+        backend (BaseBackend): a backend to compile for
```
Comment (ajavadia) [inline qiskit/wrapper/_wrapper.py]: Need to fix this in the docstring of `execute` as well.

File: test/python/aer/test_simulator_interfaces.py [hunk 2]
```diff
@@ -53,18 +52,14 @@ def test_qasm(self, qe_token, qe_url):
         circuit.cx(qr[1], qr[2])
         circuit.measure(qr, cr)
 
-        sim_cpp = 'qasm_simulator'
-        sim_py = 'qasm_simulator_py'
-        sim_ibmq = 'ibmq_qasm_simulator'
```
Comment (ajavadia) [inline test/python/aer/test_simulator_interfaces.py]: Why remove the run on the ibm_qasm_simulator in this test? This test is meant to do some cross testing between different simulators to make sure they have the same output for a given input.

Comment (jaygambetta) [reply test/python/aer/test_simulator_interfaces.py]: Because it’s in the aer folder and redone in ibmq

Comment (jaygambetta) [reply test/python/aer/test_simulator_interfaces.py]: I see I left ibmq in the import as I went between keeping and not a few times. :-)

Comment (ajavadia) [reply test/python/aer/test_simulator_interfaces.py]: oh yeah. good point.

File: qiskit/wrapper/_wrapper.py (hunk 1 above)
Comment (jaygambetta) [reply qiskit/wrapper/_wrapper.py]: done

**Pull request merged**